src/components/Carousel.tsx

```diff
@@ -22,9 +24,10 @@ export default function Carousel({
 	maxWidth,
 	withArrows = true,
 	withDots = true,
+	withFade = false,
 	carouselItems,
 	...rest
-}: CarouselProps) {
+}: FadedCarouselProps) {
```

Comment:
Can you rename the component as well please

Reply:
ReFixed to `CarouselProps` as the renaming didn't take place.